_posts/2021-02-03-episode-three-of-lets-talk-about-public-code.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Episode three of Let's talk about public code! - Sergei Maertens and Tjerk Vaags from OpenZaak"
+date: 2021-02-03
+author: Jan Ainali
+type: blogpost
+excerpt: The third episode of our podcast is available
+categories:
+  - news
+---
+
+# Episode three of Let's talk about public code!
+
+In the third episode we talked to Sergei Maertens from Maykin Media and Tjerk Vaags from Contezza.
+How can you design a process for handling Common Vulnerabilities and Exposures worked in a collaborative codebase?
+We got to know how they handled a security issue in OpenZaak and hopefully you can be inspired by it.
```

Comment:
```suggestion
We got to know how they handled a security issue in OpenZaak. Hopefully this will inspire you as you consider what your project would do in the same situation.
```